dotnet/src/webdriver/DevTools/DevToolsSession.cs

```diff
@@ -46,14 +47,15 @@ public class DevToolsSession : IDevToolsSession
         private bool isDisposed = false;
         private string attachedTargetId;
 
-        private ClientWebSocket sessionSocket;
+        private WebSocketConnection connection;
         private ConcurrentDictionary<long, DevToolsCommandData> pendingCommands = new ConcurrentDictionary<long, DevToolsCommandData>();
+        private readonly Channel<string> messageQueue;
```

Comment:
Seems `System.Threading.Channels` package is needed only for queue. I think we can use any other queue built in BCL, and remove external dependency.

Reply:
`Channel<T>` provides a far cleaner API for the type of producer/consumer pattern that we are enabling here, but I'll look into a change using a `ConcurrentQueue<T>` instead.

Reply:
I also like https://learn.microsoft.com/en-us/dotnet/api/system.collections.concurrent.blockingcollection-1?view=net-7.0 with very easier way of collection consumption:
```
foreach (var item in bc.GetConsumingEnumerable())
{
    Console.WriteLine($"Consuming: {item}");
}
```